use validator::ValidationError; 

pub fn validate_unique_username() {
    todo!()
}

pub fn validate_phone_number() -> Result<(), ValidationError> {
    todo!();
    Ok(())
}